go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/giantswarm/logging-operator
 
-go 1.19
+go 1.18
```

Comment:
why the rollback in go version?

Reply:
Oh, because our build system does not support go 1.19! :fearful: 
![image](https://github.com/giantswarm/logging-operator/assets/12008875/57100898-70c6-41e6-8137-b8c6b040ad4b)

Anyway fixing the build should not be in this "Helm chart" PR IMO.

Reply:
Fixing in https://github.com/giantswarm/logging-operator/pull/16